app/assets/stylesheets/local/work-show.scss

```diff
@@ -182,6 +182,10 @@
       // aspect ratio, cause the dimensions are so responsive. This is just
       // a rough huge limit, if it's not working out just delete this and give up.
       max-height: 300vw;
+
+      &.pdf_border {
+        border: 2px solid #050939;
+      }
```

Comment:
should we define a custom CSS mixin, so we don't need to repeat the border definition in two places?  Or find another way to not need to repeat the border definition in two places? Low priority, not a big deal if we keep repeating.